### ISSUE TITLE
Clean cookies and session each scenario

### ISSUE THREAD

**Issue by mantovanig** [garris/BackstopJS#421]
Hi,

is possible clean cookies and session for each scenario?
For example if i want repeat the login flow in each scenario.

I've tried to do this in onBefore.js:

```
module.exports = function (casper, scenario, vp) {
  
  casper.page.clearCookies();

  console.log('onBefore.js has run for '+ vp.name + '.');
};
```

But doesn't work

Thanks

**Comment by garris** [comment]
Not clearing cookies is a bug. 

Are you able to set the cookie to another value?

**Comment by mantovanig** [comment]
Ok.

And there is another solution for have a new session in each scenario?
Or i must lunch backstop('test') with one scenario at once?

Thanks

**Comment by garris** [comment]
Yes,  I think the work-around is to use one config per `onBefore` script.   Please let me know if that works for you.

**Comment by garris** [comment]
We should probably explicitly declare that as a known issue.

**Comment by garris** [comment]
closed by #442
